### DISCUSSION TITLE
Subtitle requests with Bazarr

### DISCUSSION THREAD

**Discussion by bleak183** [sct/overseerr#3687] [Feature Requests]
Just thought it would be really nice if Overseerr worked with Bazarr and users could request subtitles in different languages directly through the Overseerr UI when requesting a movie or show.

**Comment by NeoTheOne** [comment]
I was also thinking about this, maybe even integrate it with the issues tab and allow users to report unsynced subs with details for proper alignment and for the admin to "approve" the change and get it pushed to Bazarr.

**Reply by bleak183** [reply]
Great idea!
